docs/UserGuide.md

```diff
@@ -255,18 +294,19 @@ Format: `exit`
 
 ## Command summary
 
```

Comment:
Just a note here: please see if there's an extra line break under `## Command summary`, otherwise the table won't form!